src/components/ui/IconButton/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react"; 
+import { View } from "react-native";
+import Icon from "@app/components/ui/Icon";
+import { colors } from "@app/utils";
+
+interface IconButtonProps {
+  id?: string;
+  onPress: () => void; 
+}
+
+const IconButton: React.FC<IconButtonProps>  = ({id, onPress}) => {
+
+  return(
+    <View>
```

Comment:
It would be good to have a look at a thing called `TouchableOpacity` in react native. It's the component that allows you to create buttons that decrease in opacity when clicked. Add the onPress callback into that one

https://reactnative.dev/docs/touchableopacity